### PR TITLE
CI: Be explicit on compilers

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: maistra/ci/post-submit.sh [hunk 1]
```diff
@@ -19,17 +19,17 @@ gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIAL
 gcloud config set project "${GCS_PROJECT}"
 
 # Build WASM extensions first
-bazel_build //extensions:stats.wasm
-bazel_build //extensions:metadata_exchange.wasm
-bazel_build //extensions:attributegen.wasm
-bazel_build @envoy//test/tools/wee8_compile:wee8_compile_tool
+CC=clang CXX=clang++ bazel_build //extensions:stats.wasm
```
Comment (dmitri-d) [inline maistra/ci/post-submit.sh:22]: This generally looks ok, but envoy has a script (here: https://github.com/maistra/envoy/blob/maistra-2.1/bazel/setup_clang.sh) that generates build:clang config that then can be used with bazel. Would it be better to generate it and then use that instead?

**Comment by dmitri-d** [comment]
Looks good, but WDYT re: my comment. If you don't think it's worth it, consider this as an approval.

File: maistra/ci/post-submit.sh (hunk 1 above)
Comment (jwendell) [reply maistra/ci/post-submit.sh:22]: I don't think we need it. Setting these variables should be enough and gives us more flexibility...

**Review by dmitri-d — APPROVED** [review]

**Pull request merged**